types/dygraphs/index.d.ts

```diff
@@ -8,7 +8,7 @@
 /// <reference types="google.visualization" />
 
 export namespace dygraphs {
-    type DataArray = Array<Array<number | Date | null>>;
+    type DataArray = Array<Array<number | number[] | Date | null>>;
```

Comment:
Please add a corresponding test for the data format that you want to support. Ideally it would come either from the dygraphs documentation or dygraphs unit tests.